component/FilterButton/FilterButoon.styled.js

```diff
@@ -0,0 +1,17 @@
+import { styled } from "styled-components";
```

Comment:
```suggestion
import styled from "styled-components";
```
`styled` is a default export.